### ISSUE TITLE
Group data is saved in db but not available on UI in phpMyFaq 3.0.3

### ISSUE THREAD

**Issue by ushah001** [thorsten/phpMyFAQ#1778]
Group listing is not available in phpMyfaq 3.0.3 thus, making all group related functions unavailble.
This happens when I published code on my server. 
Demo given at https://demo.phpmyfaq.de/ is working as per expectation. 
**To Reproduce**
1. Login as Administrator.
2. Click on Users on left hand side.
3. Scroll down to 'Groups'
4. Add a group
5. You will see Group is added successfully message. same you can confirm in the database in table - faqgroups 
6.no group is added in groups List on Groups page.

**Expected behavior**
A new group should be added in Groups List.

**phpMyFAQ (please complete the following information):**
 - phpMyFAQ version 3.0.3
 - Database : Sql server (added as a linked server)
 - Elasticsearch yes/no

**Desktop (please complete the following information):**
 - OS: Windows 10
 - Browser - Chrome
 - Version : 83

**Additional context**
Security level is set to Medium for grouping functionality as described in documentation.

![image](https://user-images.githubusercontent.com/67292745/85334796-d852d000-b4a9-11ea-84d6-351dc33092a0.png)

![image](https://user-images.githubusercontent.com/67292745/85334858-f28cae00-b4a9-11ea-918c-429e1ea22eff.png)

![image](https://user-images.githubusercontent.com/67292745/85334895-046e5100-b4aa-11ea-9723-aeca3b616ee4.png)

**Comment by thorsten** [comment]
Looks like the query is not working on MS SQL.

**Comment by thorsten** [comment]
Could you please try this query:

`SELECT fg.group_id FROM faqgroup fg LEFT JOIN faquser_group fug ON fg.group_id = fug.group_id WHERE fug.user_id = 1;`

**Comment by ushah001** [comment]
Hey I figured out the solution. 

Issue was with the configuration - **Enable Category restrictions** 

If **Enable Category restrictions** is False and Group is added - it doesn't show on the page even thought it is present in the db. 
I made it true and it worked. 

Not sure if I missed this configuration from documentation or it's a bug. 

![image](https://user-images.githubusercontent.com/67292745/92136087-9adbbe80-edd9-11ea-8e62-6e0dfe3b695b.png)
